### ISSUE TITLE
Accept HTML Color code

### ISSUE THREAD

**Issue by kuldeepdhaka** [openscad/openscad#2100]
Please accept \#RRGGBB (and \#RRGGBBAA will not be a big either if the non alpha is implemented)

People (including me) who have worked with the HTML color code will find it very handy.

I dont think this is supported because nothing mentioned in documentation
https://en.wikibooks.org/wiki/OpenSCAD_User_Manual/Transformations#color

and if i try to use, result in
```
WARNING: Color name "#b5a642" unknown. Please see
WARNING: http://en.wikipedia.org/wiki/Web_colors
```

**Comment by MichaelAtOz** [comment]
Thanks, I needed a challenge.
[rgb.txt](https://github.com/openscad/openscad/files/1250754/rgb.txt)

**Comment by Corona688** [comment]
This is a terrific idea.

**Comment by justbuchanan** [comment]
Closed as of #2134.
